worker/sshserver/export_test.go

```diff
@@ -0,0 +1,8 @@
+// Copyright 2025 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package sshserver
+
+var (
+	NewSSHServerListener = newSSHServerListener
```

Comment:
We don't use export_tests. You need to either just expose it, or use white box testing.